pytest-embedded-idf/tests/test_idf.py

```diff
@@ -12,6 +14,15 @@
 )
 
 
+@pytest.fixture
+def remove_serial(monkeypatch):
+    monkeypatch.setitem(sys.modules, 'pytest-embedded-serial', None)
```

Comment:
the module name can't be `pytest-embedded-serial` right? should be `pytest_embedded_serial`?